src/pages/overview/pdf-services-api/howtos/webhook-notification.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Webhook Notification support for Adobe PDF Services APIs | Adobe PDF Services
+---
+# **Webhook Notification support for Adobe PDF Services APIs**
```

Comment:
Webhook Notification Support for Adobe PDF Services APIs

Adobe PDF Service APIs now offer support for Webhook notifications. Webhooks are plugins designed to facilitate the receipt of job completion notifications. This new feature eliminates the need for clients to manually invoke the GET Status API to obtain job completion status and results. Instead, they will receive automatic notifications containing the job status and result.

To leverage the power of webhooks, clients are required to include the "notifiers" key in the request body when making a call to the "Submit Job" API. The "notifiers" key is designed to receive a list of objects, with each object corresponding to a specific webhook. Each notifier object is composed of two essential fields: "type" and "data." Here, we provide detailed information about these fields and offer a sample request:

type: This field specifies the type of notifier to be invoked. Currently, the only supported notifier type is [CALLBACK](https://chat.openai.com/c/16347e54-6885-4fa6-9eb3-93a29ad2381d#callback-notifier).

data: This field contains the data required for the webhook. The specific data may vary depending on the type of notifier in use.

For more information on the "CALLBACK" notifier, please refer to the [Callback Notifier](https://chat.openai.com/c/16347e54-6885-4fa6-9eb3-93a29ad2381d#callback-notifier) section.